completion.py

```diff
@@ -44,7 +47,30 @@ def get_completion(clones_dir: str, repo: str) -> tuple[float, 'TranslatorsData'
             hide_reserved=False,
             api_url='',
         ).completion
-    return completion, translators_data, branch
+
+    try:
```

Comment:
```suggestion
    if completion:
```
I suggest to calculate the change only if the completion is greater than 0. This way we can get rid of handling the case when there's no repository with the try-except.